fs/accounting.go

```diff
@@ -21,6 +22,7 @@ var (
 	tokenBucketMu   sync.Mutex // protects the token bucket variables
 	tokenBucket     *tb.Bucket
 	prevTokenBucket = tokenBucket
+	statsBps        = pflag.BoolP("statsbps", "", false, "Show stats output in bits per second")
```

Comment:
I'm not keen on the name - `--statsbps` is that bytes/s or bit/s?  How about `--stats-unit bits` or `--stats-unit bytes`?

I think you should define this in `cmd/cmd.go`, next to the `--stats` definition, and parse it in `initConfig` into a flag.

Then add an additional parameter to `Stats.Log` and `Stats.String` indicating the units.  This is called from `cmd/cmd.go`. This will need a little bit of tweaking of stuff to make work.